pysass.cpp

```diff
@@ -404,6 +435,116 @@ static void _add_custom_functions(
     sass_option_set_c_functions(options, fn_list);
 }
 
+Sass_Import_List _call_py_importer_f(
+        const char* path,
+        Sass_Importer_Entry cb,
+        struct Sass_Compiler* comp
+) {
+    PyObject* pyfunc = (PyObject*)sass_importer_get_cookie(cb);
+    PyObject* py_path = PyUnicode_FromString(path);
+    PyObject* py_args = PyTuple_New(1);
+    PyObject* py_result = NULL;
+    Sass_Import_List sass_imports = NULL;
+    Py_ssize_t i;
+    
+    PyTuple_SetItem(py_args, 0, py_path);
+    
+    if (!(py_result = PyObject_CallObject(pyfunc, py_args))) {
```

Comment:
Can probably use https://docs.python.org/2/c-api/object.html#c.PyObject_CallFunction instead:

```c
PyObject_CallObject(pyfunc, PySass_IF_PY3("y", "s"), py_path)
```

This'll avoid the complexity of the tuple (the only reason I didn't do this for custom functions is there are a variable number of arguments)